install/install.sh

```diff
@@ -162,11 +162,16 @@ echo "Going to install blueos-docker version ${VERSION}."
 echo "Downloading and installing udev rules."
 curl -fsSL $ROOT/install/udev/100.autopilot.rules -o /etc/udev/rules.d/100.autopilot.rules
 
-echo "Disabling automatic Link-local configuration in dhcpd.conf."
-# delete line if it already exists
-sed -i '/noipv4ll/d' /etc/dhcpcd.conf
-# add noipv4ll
-sed -i '$ a noipv4ll' /etc/dhcpcd.conf
+if [ -e /etc/dhcpcd.conf ]
```

Comment:
Would use `-f` here to check if it is truly a file and not a directory (of course very unlikely).

Reply:
Done. Sorry I rebased instead of a simple commit - my setup is being weird

Reply:
No problem.